src/ngraph/frontend/onnx_import/op/squeeze.cpp

```diff
@@ -29,8 +31,10 @@ namespace ngraph
                 NodeVector squeeze(const Node& node)
                 {
                     auto data = node.get_ng_inputs().at(0);
-                    auto axes = AxisVector{node.get_attribute_value<std::vector<std::size_t>>("axes", {})};
-                    return {std::make_shared<ngraph::op::Squeeze>(data, axes)};
+                    auto axes = node.get_attribute_value<std::vector<std::int64_t>>("axes", {});
+                    auto axes_node = std::make_shared<ngraph::op::Constant>(
+                        element::u64, Shape{axes.size()}, axes);
```

Comment:
Is it intended to use `unsigned int` as element type for `axes_node` while `axes` is `int`?

Reply:
If negative values will come as an input this will cause errors. Please handle negative values or according to [doc](https://github.com/onnx/onnx/blob/master/docs/Operators.md#attributes-69) restrict to non-negative values.